models/model-topics.js

```diff
@@ -7,3 +7,14 @@ exports.selectTopic = () => {
     
 }
 
+exports.selectArticle = () => {
```

Comment:
I'd make this function name plural eg `selectArticles` the naming at the moment suggests this is only selecting a single article.